deployment/README.md

```diff
@@ -0,0 +1,56 @@
+# Deployments
+
+## k8s Deployments
+
+### Production, Staging & Develop
+
+- lexbox
+  - .net API (GQL API, REST API, hg proxy etc. (see [backend README](../backend/README.md)))
+  - OTEL collector
+- db (PostgresDB)
+- hg
+  - hg
+  - hg-resumable
+- ui (SvelteKit app)
+
+### Local development
+
+Same as above plus:
+
+- lexbox
+  - maildev
```

Comment:
said db-migrations init container is removed in local dev, migrations are done at app startup there.